README.md

```diff
@@ -3,6 +3,9 @@
 [![Build Status](https://travis-ci.org/databricks/spark-redshift.svg?branch=master)](https://travis-ci.org/databricks/spark-redshift)
 [![codecov.io](http://codecov.io/github/databricks/spark-redshift/coverage.svg?branch=master)](http://codecov.io/github/databricks/spark-redshift?branch=master)
 
+## Disclaimer
+This is fork version from Databricks's spark-redshift repository. Our custom changes only tested with Spark **2.4.0** version. These custom changes may not be worked with older version of Spark
```

Comment:
I believe Yelp's tend to only test against Spark 2.4 with Mesos. We should make sure we establish some testing procedure with EMR as well (I think Udemy uses EMR). We can make it different ticket to keep this work progressing.

Reply:
Yes, I'm focusing on getting  a baseline to work for Yelp, do we need testing on EMR internally? I want to publicize this as soon as possible so to get some help from the community. If anyone needs testing against EMR I'm happy to accept PRs and merge them in.